sim:
  gui: True
  sleep: True
  scene: pile

urdf_root: ../urdf_zoo/models/blocks
scaling:
  low: 0.8
  high: 1.4 
object_count_lambda: 3
view_count_lambda: 1
scene_grasp_count: 200

metric: robust_physics

robust_physics:
  uncertainty:
    xyz: [0.004, 0.004, 0.004]
    rpy: [0.02, 0.02, 0.02]
  sample_count: 6
